api/src/main/java/org/openmrs/module/referencedemodata/patient/DemoPatientGenerator.java

```diff
@@ -187,8 +214,9 @@ public void createDemoPatients(int patientCount) {
 		
 		for (int i = 0; i < patientCount; i++) {
 			Patient patient = createDemoPatient(ps, patientIdentifierType, rootLocation);
-			log.info("created demo patient: {} {} {}", patient.getPatientIdentifier(), patient.getGivenName(),
+			String info = String.format("created demo patient: %s %s %s", patient.getPatientIdentifier().toString(), patient.getGivenName(),
```

Comment:
?

Maybe this needs to be:

```java
log.info("created demo patient: {} {} {}", new Object[] { patient.getPatientIdentifier(), patient.getGivenName(), patient.getFamilyName() });
```

But this seems wrong (`toString()` is redundant; using the formatting options from SLF4J defers the string interpolation so that it doesn't happen if the message doesn't need to be logged, etc.)